public/tier1/utlvector.h

```diff
@@ -1161,6 +1161,11 @@ template<class T> class CUtlVectorAutoPurge : public CUtlVector< T, CUtlMemory<
 class CUtlStringList : public CUtlVectorAutoPurge< char *>
 {
 public:
+	~CUtlStringList()

```

Comment:
Should CUtlVectorAutoPurge's destructor be virtual instead?

Reply:
Not really since the use case for this class doesn't infer that you should cast it away to a base class and free that instead. (In which case only the base class destructor would've been called and using virtual would solve the issue, otherwise it's fine. Also I don't think any of these do have virtual tables in the game binaries to justify that change)